src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbolBase.cs

```diff
@@ -60,7 +63,8 @@ internal abstract class SourceOrdinaryMethodSymbolBase : SourceOrdinaryMethodOrU
             bool isExplicitInterfaceImplementation = methodKind == MethodKind.ExplicitInterfaceImplementation;
             var isMetadataVirtualIgnoringModifiers = isExplicitInterfaceImplementation && (declarationModifiers & DeclarationModifiers.Static) == 0;
 
-            this.MakeFlags(methodKind, declarationModifiers, returnsVoid, isExtensionMethod: isExtensionMethod, isNullableAnalysisEnabled: isNullableAnalysisEnabled, isMetadataVirtualIgnoringModifiers: isMetadataVirtualIgnoringModifiers);
+            this.MakeFlags(methodKind, declarationModifiers, returnsVoid, isExpressionBodied: isExpressionBodied, isExtensionMethod: isExtensionMethod, isNullableAnalysisEnabled: isNullableAnalysisEnabled, isMetadataVirtualIgnoringModifiers: isMetadataVirtualIgnoringModifiers);
+            flags.SetOrdinaryMethodFlags(refKind, hasBody, isVarArg);
```

Comment:
>flags.SetOrdinaryMethodFlags(refKind, hasBody, isVarArg); [](http://example.com/codeflow?start=12&length=57)

It looks like we are simply passing parameters as is. Is it possible to pass them to base constructor and get rid of this helper?

Reply:
I guess, passing to MakeFlags would be more appropriate

Reply:
yes, but at the cost of everyone passing this along.  i'm not sure to what level we would prefer impacting all ocde, or just the specific code that needs the new facilities.  My tendancy now in the compiler code is to minimize the scope of a change to have better chances of not falling afoul of refactorings being considered too broad.

Reply:
> yes, but at the cost of everyone passing this along. 

At the moment I cannot think of any good reason for splitting flag initialization in two steps (lazy flags is an exception). In fact, splitting them is bug prone, since some existing or new code might fail to initialize all flags explicitly.

Reply:
the good reason was simply that it minimized churn.  If churn is acceptable, i'm happy to make more invasive changes like that.  That's honestly my preference when it comes to refactorings, in order to have safer and more sound code.  I'll work on the changes later.

Reply:
>  If churn is acceptable,

The churn primarily related to the affected flags is acceptable. Not an arbitrary churn.

Reply:
To clarify, changes around other flags will be undesired in context of this PR, in my opinion